doc/JTS_Version_History.html

```diff
@@ -15,22 +15,77 @@ <h1 style='text-align:center; color:darkblue;'>
 This document lists the change history of release versions of the
 <span style='color:darkblue;'><b>JTS Topology Suite</b></span>.
 <p>
-Further information on JTS can be found at <b>Tsusiat Software's
-<a href='http://tsusiatsoftware.net/jts/main.html'>JTS pages</a></b>.
+The current JTS distributions can be obtained at the
+<a href='https://github.com/locationtech/jts/releases'><b>Github JTS site</b></a>.
```

Comment:
do you want to use the project website at http://locationtech.org/projects/technology.jts